DEVELOPER_GUIDE.md

```diff
@@ -436,13 +433,13 @@ All transport headers found in the client request and backend response will be s
     </LimitTransportHeaders>
    ```
 
-Only the intercept of headers mentioned and present in the transport headers will be sent to ASE in both sideband calls.
+Only the intercept of headers mentioned and present in the transport headers are sent to ASE in both sideband calls.
 
-*If there is a transport header which changes with each request, it is essential to use this feature and drop that header. Otherwise feature will not be useful in the async and hybrid modes.*
+*If there is a transport header which changes with each request, it is essential to use this feature and drop that header. Otherwise, this feature will not be useful when working with async and hybrid modes.*
```

Comment:
Shall we change "Otherwise, this feature will not be useful" to "Otherwise, this extension will not be useful"

Reply:
Fixed